site-docker.yml.sample

```diff
@@ -50,16 +50,16 @@
   roles:
     - { role: ceph-defaults,
         tags: [with_pkg, fetch_container_image],
-        when: "(((containerized_deployment | bool) and not (is_atomic | bool) and not (inventory_hostname in groups.get('clients', []))) or ((inventory_hostname == groups.get('clients', [''])|first) and (containerized_deployment | bool) and not (is_atomic | bool)))" }
+        when: "(containerized_deployment | bool) and not (is_atomic | bool) and (not (inventory_hostname in groups.get('clients', [])) or (inventory_hostname == groups.get('clients', [''])|first))" }
```

Comment:
These can also be listed out to increase legibility. All list elements in the `when` field get joined with `and`.


```yaml
- role: ceph-defaults
  tags: [with_pkg, fetch_container_image]
  when:
    - containerized_deployment | bool
    - not (is_atomic | bool)
    - (not (inventory_hostname in groups.get('clients', [])) or (inventory_hostname == groups.get('clients', [''])|first))
```

Reply:
@samdoran thanks!